doc/examples.rst

```diff
@@ -142,3 +146,31 @@ Example of the use of transformations to introduce redundant computation,
 split synchronous halo exchanges into asynchronous exchanges (start and
 stop) and move the starts of those exchanges in order to overlap them
 with computation.
+
+Example 12
+^^^^^^^^^^
+
+Example of applying code extraction to Nodes in an Invoke Schedule:
+
+.. code-block:: bash
+
+  > psyclone -nodm -s ./extract_nodes.py \
+      gw_mixed_schur_preconditioner_alg_mod.x90
+
+or to a Kernel in an Invoke after applying transformations:
+
+.. code-block:: bash
+
+  > psyclone -nodm -s ./extract_kernel_with_transformations.py \
+      gw_mixed_schur_preconditioner_alg_mod.x90
+
+For now it only inserts comments in appropriate locations while the
+the full support for code extraction is being developed.
+
+This example also contains a Python helper script ``find_kernel.py``
+which displays useful information for Kernel extraction: names and
```

Comment:
suggest "...displays the names and Schedules of Invokes containing call(s) to the specified Kernel:"

Reply:
Done